### DISCUSSION TITLE
Example of http post request

### DISCUSSION THREAD

**Discussion by 2005m** [vlang/v#14649] [Ideas]
Hi,
Would you be able to provide an example of http post request with the following features?

Content-type = "application/xml;charset=UTF-8"
Authorization="key"
Accept="application/xml;charset=UTF-8,text/csv"
User-Agent= "John"
Require ssl verification= false
Proxy= none

Thank you

**Comment by hungrybluedev** [comment]
See https://github.com/hungrybluedev/klonol/blob/main/github/github.v#L8 for example and https://modules.vlang.io/net.http.html#Request for reference.

**Comment by 2005m** [comment]
@hungrybluedev , thank you. These links are helpful. I am wrong or is ssl not implemented? Also I did not quite understand how to ignore any proxy setting in the case environment variable like `HTTP_PROXY` is set. Thank you.

**Comment by JalonSolov** [comment]
http vs https is handled automatically in the code.  There are no separate calls.

**Comment by 2005m** [comment]
Maybe PR #12276 would solve it?
